index.js

```diff
@@ -1,21 +1,18 @@
-var getYouTubeID = require('get-youtube-id');
```

Comment:
This file can't use ES6 since it's executed in Node.

The process function should not be an arrow function, because `this.output` will be undefined.